site/blog/tutorials/iasql-on-ec2.mdx

```diff
@@ -203,3 +203,138 @@ WHERE id = (SELECT id
+:::
+
+Now that we have created our hosted zone, we need to set our domain's nameservers to the ones that Route53 provides us. For each public hosted zone that you create, Amazon Route53 [automatically creates](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/SOA-NSrecords.html) a name server (NS) record and a start of authority (SOA) record. So we should log into the domain registrar, and set the name server values to the four name server values that Route53 has set. To get those values, we can simply run a `SELECT` query:
+```sql
+SELECT record as name_server
+FROM resource_record_set
+WHERE parent_hosted_zone_id = (SELECT id FROM hosted_zone WHERE domain_name = '<your-domain>')
+  AND record_type = 'NS';
+```
+After updating the name servers in the domain settings, we need to wait for those values to be propagated. This can happen in minutes, or even up to 48 hours depending on the registrar.
+
+#### Issue An SSL Certificate For Our Domain
+Having the domain setup in Route53, it's easy to use IaSQL's `certificate_request` RPC to issue a new certificate for it. If you try to go with this procedure through AWS console, you'll need to manually add the CNAME records from the AWS ACM to your hosted zone. But this RPC is implemented in a way that automagically [sets the DNS records](https://github.com/iasql/iasql/blob/ed1b1ef92ceaa92e7345449992c35a56b62fe571/src/modules/aws_acm/rpcs/request.ts#L177-L188), [waits for the certificate to be issued](https://github.com/iasql/iasql/blob/ed1b1ef92ceaa92e7345449992c35a56b62fe571/src/modules/aws_acm/rpcs/request.ts#L197), and then [removes those validation DNS records](https://github.com/iasql/iasql/blob/ed1b1ef92ceaa92e7345449992c35a56b62fe571/src/modules/aws_acm/rpcs/request.ts#L200).
+
+```sql
+-- SELECT iasql_install('aws_acm'); -- no need for this because aws_route53 depends on it and it's automatically installed
+SELECT certificate_request('<your-domain>', 'DNS', default_aws_region(), '{}');
+
+-- inspect the new certificate
+SELECT *
+FROM certificate
+WHERE domain_name = '<your-domain>' AND status = 'ISSUED';
+```
+
+The second query confirms that we have a new certificate for our domain, and it's ready to be used (status = `ISSUED`). Now we're ready to launch our load balancer, listen for HTTPS traffic with this new certificate, and forward the incoming traffic to our EC2 instance's 9876 port.
+
+#### Spawn An Elastic Load Balancer
+To set up an elastic load balancer, we need some kinds of resources:
+- `load_balancer` itself
+- A `target_group` pointing to our EC2 instance on port 9876
+- A `listener` on port 443 of the load balancer
+
+<img src={'/img/iasql-on-ec2/load-balancer-parts.png'} style={{maxWidth: 300}} />
```

Comment:
I don't this zoomed-in diagram with so few moving pieces is adding any value clarity. same for the one that is already in the blog post further up. we should remove these and just keep the big, high-level one